src/styles/blocks/product.scss

```diff
@@ -0,0 +1,63 @@
+.product {
+  box-sizing: border-box;
+  display: flex;
+  width: $card-width;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  padding: 32px 16px 17px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+  transition-timing-function: ease-in-out;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+  &__image {
+    margin: auto;
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &:hover
+  &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin: 4px 0 9px;
+    font-weight: 400;
```

Comment:
`400` is a default value for `font-weight`, so you can remove this and it won't affect anything